packages/calling/src/CallingClient/CallingClient.test.ts

```diff
@@ -254,31 +257,156 @@ describe('CallingClient Tests', () => {
 
       callingClient = await createClient(webex, {logger: {level: LOGGER.INFO}});
 
-      expect(handleErrorSpy).toBeCalledOnceWith(failurePayload, expect.anything(), {
-        file: CALLING_CLIENT_FILE,
-        method: 'getMobiusServers',
+      expect(webex.request).nthCalledWith(1, {
+        ...getMockRequestTemplate(),
+        uri: 'https://mobius-us-east-1.prod.infra.webex.com/api/v1/calling/web/myip',
+        method: 'GET',
       });
 
-      expect(webex.request).toBeCalledOnceWith({
+      expect(webex.request).nthCalledWith(2, {
         ...getMockRequestTemplate(),
-        uri: `${uri}myip`,
+        uri: 'https://mobius-ca-central-1.prod.infra.webex.com/api/v1/calling/web/myip',
         method: 'GET',
       });
 
-      expect(callingClient.primaryMobiusUris).toEqual([uri]);
-
-      expect(webex.request).nthCalledWith(1, {
+      expect(webex.request).nthCalledWith(3, {
+        ...getMockRequestTemplate(),
+        uri: 'https://mobius-eu-central-1.prod.infra.webex.com/api/v1/calling/web/myip',
         method: 'GET',
+      });
+
+      expect(webex.request).nthCalledWith(4, {
         ...getMockRequestTemplate(),
-        uri: `${uri}myip`,
+        uri: 'https://mobius-ap-southeast-2.prod.infra.webex.com/api/v1/calling/web/myip',
+        method: 'GET',
+      });
+
+      expect(handleErrorSpy).toBeCalledWith(failurePayload, expect.anything(), {
+        file: CALLING_CLIENT_FILE,
+        method: 'getMobiusServers',
       });
 
+      expect(callingClient.primaryMobiusUris).toEqual([
+        `${callingClient['mobiusHost']}${URL_ENDPOINT}`,
+      ]);
+
       expect(warnSpy).toBeCalledWith(
-        'Error in finding Mobius Servers. Will use the default URL.',
+        `Couldn't resolve the region and country code. Defaulting to the catalog entries to discover mobius servers`,
         ''
       );
     });
 
+    it('verifying entries of eu mobius clusters', async () => {
+      const mockEUServiceHosts = [
```

Comment:
can we move these to a fixtures file so we don't have so many extra lines in the test file?

Reply:
done